CHANGELOG.rst

```diff
@@ -4,6 +4,8 @@ Changelog
 in development
 --------------
 
+* Add json_ecsape Jinja filter for escaping JSON strings
```

Comment:
Would be nice to reference #3447 or #3480 in the message, as well as include the type of record: is it improvement, new feature, bugfix.
See previous messages in the changelog for consistency.